mix.exs

```diff
@@ -1,59 +1,66 @@
 defmodule UeberauthCognito.MixProject do
   use Mix.Project
 
+  @source_url "https://github.com/mbta/ueberauth_cognito"
   @version "0.3.0"
-  @url "https://github.com/mbta/ueberauth_cognito"
 
   def project do
     [
       app: :ueberauth_cognito,
+      name: "Ueberauth Cognito",
       version: @version,
       elixir: "~> 1.10",
       start_permanent: Mix.env() == :prod,
-      deps: deps(),
       test_coverage: [tool: LcovEx],
-      description: description(),
       package: package(),
-      name: "Ueberauth AWS Cognito Strategy",
-      source_url: @url,
-      docs: [
-        main: "readme",
-        extras: ["README.md"]
-      ]
+      deps: deps(),
+      docs: docs()
     ]
   end
 
-  # Run "mix help compile.app" to learn about applications.
   def application do
     [
       extra_applications: [:logger]
     ]
   end
 
-  # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
       {:lcov_ex, "~> 0.2", only: [:dev, :test], runtime: false},
       {:hackney, "~> 1.0"},
       {:jason, "~> 1.0"},
       {:jose, "~> 1.0"},
       {:ueberauth, "~> 0.7"},
-      {:ex_doc, "~> 0.19", only: :dev, runtime: false}
+      {:ex_doc, ">= 0.0.0", only: :dev, runtime: false}
```

Comment:
Can you leave this as `~>`? It's fine to bump to `"~> 0.25"`, though.

Reply:
Done, but I've bumped to latest greatest, `0.26`.